src/platform/windows/window.rs

```diff
@@ -33,6 +33,8 @@ use platform::platform::monitor::get_available_monitors;
 use platform::platform::raw_input::register_all_mice_and_keyboards_for_raw_input;
 use platform::platform::util;
 
+const WS_RESIZABLE: LONG = (winuser::WS_SIZEBOX | winuser::WS_MAXIMIZEBOX) as LONG;
```

Comment:
Isn't it more semantically appropriate to store this as a `DWORD`? This ought to be the canonical form for this piece of data.

Reply:
Sometimes it was used as a LONG and sometimes it was used as a DWORD so I didn't really know which to use, will switch it out